### DISCUSSION TITLE
Rmd relative path

### DISCUSSION THREAD

**Discussion by AndrewSkelton** [posit-dev/positron#4688] [Q&A]
Hi, 

Typical behavior in RStudio was that when an Rmd file was open, paths in the notebook were relative to where the file was located, this doesn't seem to be the same in Positron. Is there a setting to enable that?

Thanks, 

Andrew

**Comment by juliasilge** [comment]
Thanks for this feedback! I've opened https://github.com/posit-dev/positron/issues/4692 to prioritize and track this feature.
